linux/meshd-nl80211.c

```diff
@@ -429,13 +429,14 @@ static int handle_wiphy(struct mesh_node *mesh, struct nl_msg *msg, void *arg)
 	return 0;
 }
 
-static int new_unauthenticated_peer(struct netlink_config_s *nlcfg,
-                                    unsigned char *peer, struct info_elems *elems)
+static int add_station(struct netlink_config_s *nlcfg,
```

Comment:
Would you object to renaming `add_station` to  `add_unauthenticated_sta`?  That clarifies the intent of the function: a new station is added to the kernel, but it is not authenticated yet.